solr_plugins/Bibelstellensuche/src/de/uni_tuebingen/ub/ixTheo/bibleRangeSearch/BibleRangeParser.java

```diff
@@ -0,0 +1,45 @@
+package de.uni_tuebingen.ub.ixTheo.bibleRangeSearch;
+
+import org.apache.lucene.search.FilteredQuery;
+import org.apache.lucene.search.Query;
+import org.apache.solr.common.params.SolrParams;
+import org.apache.solr.request.SolrQueryRequest;
+import org.apache.solr.search.QParser;
+import org.apache.solr.search.SyntaxError;
+
+public class BibleRangeParser extends QParser {
+	private final static String QUERY_SEPERATOR = " ";
+	private final static String DB_FIELD_SEPERATOR = ",";
+
+	private Query innerQuery;
+
+	public BibleRangeParser(String qstr, SolrParams localParams,
+			SolrParams params, SolrQueryRequest req) {
```

Comment:
Please indent this such that the first non-space character lines up with the first character after the opening brace above.